### PR TITLE
Adding Drupal status report validation to catch errors locally and on CI

### PR DESCRIPTION
**Pull request by davidferlay**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -162,7 +162,25 @@ test:drupalcheck:
...
+  - pwd
+  - ls -lah
+  - ls -lah scripts/makefile/
+  - make statusreportval
+  tags:
+  - review-apps-3
+  dependencies:
+  - deploy:review
+
+logs:watchdog:
```
Comment (andypost) [inline .gitlab-ci.yml]: There's no logs stage defined for pipeline

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -162,7 +162,25 @@ test:drupalcheck:
...
+  environment:
+    url: http://${CI_ENVIRONMENT_SLUG}.${REVIEW_DOMAIN}
+    name: review/$CI_COMMIT_REF_NAME
+    on_stop: stop_review
+  script:
+  - echo "Starting job script in ${BUILD_DIR}"
+  - cd ${BUILD_DIR}
+  - pwd
+  - ls -lah
+  - ls -lah scripts/makefile/
```
Comment (andypost) [inline .gitlab-ci.yml]: Lines 174-176 better to remove

File: .gitlab-ci.yml (hunk 1 above)
Comment (davidferlay) [reply .gitlab-ci.yml]: Yes there is https://github.com/skilld-labs/skilld-docker-container/blob/1102d0cf21b0d4521146af54e39ac2c985afc705/.gitlab-ci.yml#L14

File: .gitlab-ci.yml (hunk 2 above)
Comment (davidferlay) [reply .gitlab-ci.yml]: Fixed

**Pull request merged**